external/harvestPackages/harvestPackages.props

```diff
@@ -2,19 +2,19 @@
 <!-- The versions can be updated by running UpdateToLatestStablePackages target in harvestPackages.depproj -->
 <ItemGroup>
 <PackageReference Include="Microsoft.CSharp">
  <Version>4.5.0</Version>
</PackageReference>
-<PackageReference Include="Microsoft.VisualBasic">
  <Version>10.2.0</Version>
</PackageReference>
+<PackageReference Include="Microsoft.VisualBasic">
  <Version>10.3.0</Version>
</PackageReference>

```

Comment:
What are all of these ^M instances?

Reply:
Those are added by the target that automatically updates this file.

Reply:
I guess something to do with the text format when writing to the file.

Reply:
Looks like they were present before as well: https://github.com/dotnet/corefx/blob/release/2.1/external/harvestPackages/harvestPackages.props

Reply:
It'd be nice if we could fix the tool to not do that :)

Reply:
Seems like it's coming from here:
https://github.com/dotnet/corefx/blob/905e3716483755a5568ba918198e195bc5627083/external/harvestPackages/harvestPackages.depproj#L37
and specifically that `&#13;`.

Reply:
@stephentoub let's do that in a separate PR, @mmitche needs this in ASAP so he can spin a servicing build for 2.1

Reply:
Yup, totally fine.  It just jumped out at me (in large part because GitHub highlights all of them in red ;)